pokecli.py

```diff
@@ -504,7 +514,8 @@ def task_configuration_error(flag_name):
             """.format(flag_name))
 
     old_flags = ['mode', 'catch_pokemon', 'spin_forts', 'forts_spin', 'hatch_eggs', 'release_pokemon', 'softban_fix',
-                'longer_eggs_first', 'evolve_speed', 'use_lucky_egg', 'item_filter', 'evolve_all', 'evolve_cp_min', 'max_steps']
+                'longer_eggs_first', 'evolve_speed', 'use_lucky_egg', 'item_filter', 'evolve_all', 'evolve_cp_min',
```

Comment:
We don't want to add it here because this will throw saying that it doesn't exist anymore. You should just check for walk being set, print a message that walk_min and walk_max are used now instead, and then the bot should be able to keep running